starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -717,6 +693,44 @@ sub check_dither {
             }
         }
     }
+
+
+
+
+    # For eng obs, don't have OR to specify dither, so stop before doing vs-OR comparisons
+    if ( $self->{obsid} =~ /^\d*$/){
+	return if ($self->{obsid} >= $ER_MIN_OBSID);
+    }
+
+    # Get the OR value of dither and compare if available
+    my $bs_val = $dither->{state};
+    my $or_val;
+    if (defined $self->{DITHER_ON}){
+        $or_val = $dthr_cmd{$self->{DITHER_ON}};
+        # ACA-002
+        push @{$self->{warn}}, "$alarm Dither mismatch - OR: $or_val != Backstop: $bs_val\n"
```

Comment:
When getting into a section of code, you should start making a habit of replacing these post-fix conditionals with `if (test) {do something}`.  Especially for a long one like this it is much more difficult to read because the conditional is almost hidden instead of being up-front like the way I think.